packages/ui/src/ui/pages/query-tracker/Plan/GraphEditor/helpers/getOperationType.ts

```diff
@@ -0,0 +1,36 @@
+import {OperationType} from '../enums';
+import {ProcessedNode} from '../../utils';
+
+export const getOperationType = (
+    operationName: string,
+    type: ProcessedNode['type'],
+): OperationType => {
+    if (type !== 'op') {
+        return OperationType.Table;
+    }
+
+    const loverCaseName = operationName.toLowerCase().trim();
+    if (loverCaseName.includes('merge')) {
```

Comment:
Do we really need such transformation? Why is operationName treated as operationType?